org.gnome.Geary.yaml

```diff
@@ -239,9 +246,8 @@ modules:
     config-opts:
       - "-Dprofile=release"
     sources:
-      - type: archive
-        url: https://download.gnome.org/sources/geary/40/geary-40.0.tar.xz
```

Comment:
The 43 release tarball is now available, probably better to keep using that than the git ?

Reply:
I plan to release fixes to flathub as soon I fix them in Geary but I do not plan to release a sub release as soon I fix a bug.

That's why I use git + branch + commit

Reply:
I plan to release fixes to flathub as soon as  I fix them in Geary but I do not plan to release a sub release as soon I fix a bug.

That's why I use git + branch + commit

Reply:
Nevermind I just can't read... I should sleep more ^^